bower.json

```diff
@@ -27,21 +28,45 @@
     "tests"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.3.0",
-    "iron-validatable-behavior": "PolymerElements/iron-validatable-behavior#^1.0.0",
-    "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#^1.0.0",
-    "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0",
-    "paper-ripple": "PolymerElements/paper-ripple#^1.0.0",
-    "paper-styles": "PolymerElements/paper-styles#^1.1.0"
+    "polymer": "Polymer/polymer#^1.0 || ^2.0",
+    "iron-validatable-behavior": "PolymerElements/iron-validatable-behavior#^1.0 || ^2.0",
+    "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#^1.0 || ^2.0",
+    "paper-behaviors": "PolymerElements/paper-behaviors#^1.0 || ^2.0",
+    "paper-ripple": "PolymerElements/paper-ripple#^1.0 || ^2.0",
+    "paper-styles": "PolymerElements/paper-styles#^1.0 || ^2.0"
   },
   "devDependencies": {
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
-    "web-component-tester": "^4.0.0",
-    "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
-    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0.0",
-    "iron-test-helpers": "PolymerElements/iron-test-helpers#^1.0.0"
+    "test-fixture": "PolymerElements/test-fixture#^1.0 || ^2.0",
+    "iron-component-page": "PolymerElements/iron-component-page#^1.0 || ^2.0",
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0 || ^2.0",
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0 || ^2.0",
+    "iron-test-helpers": "PolymerElements/iron-test-helpers#^1.0 || ^2.0",
+    "web-component-tester": "Polymer/web-component-tester#^6.0.0",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0"
   },
-  "main": "paper-tristate-checkbox.html"
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "polymer": "Polymer/polymer#^1.9.3",
```

Comment:
This needs to match the lower case of the or above

Reply:
done

Reply:
Meaning if this is dependent on ^1.9.3 it means it should have the same requirement above.